src/clojure_tic_tac_toe/game_handler.clj

```diff
@@ -1,9 +1,12 @@
 (ns clojure-tic-tac-toe.game_handler
   (:require [clojure-tic-tac-toe.board :as board]
+            [clojure-tic-tac-toe.input_output :as io]
             [clojure-tic-tac-toe.win_checker :as win-checker]))
 
-(def starting-game-state
-  {:board board/empty-board, :player :X, :finished? false, :winner false})
+(defn create-game-state
+  [move-strategies]
+  {:board board/empty-board, :player :X, :finished? false, :winner false,
```

Comment:
we could use spacing here like you did below

Reply:
Changed

Reply:
Done